src/Humanizer/EnumHumanizeExtensions.cs

```diff
@@ -37,6 +58,43 @@ public static string Humanize(this Enum input)
             return caseName.Humanize();
         }
 
+        /// <summary>
+        /// Checks whether the given enum is to be used as a bit field type.
+        /// </summary>
+        /// <param name="input">An enum</param>
+        /// <returns>True if the given enum is a bit field enum, false otherwise.</returns>
+        private static bool IsBitFieldEnum(Enum input)
+        {
+            var type = input.GetType();
+
+            var hasFlagsAttribute = type.GetTypeInfo().GetCustomAttribute(typeof(FlagsAttribute)) != null;
+            var underlyingTypeIsInt = Enum.GetUnderlyingType(type) == typeof(int);
+
+            return hasFlagsAttribute && underlyingTypeIsInt;
+        }
+
+        /// <summary>
+        /// Returns true if the given enum instance can map exactly to one of the enum's constants.
+        /// This doesn't always happen when using BitField enums.
+        /// </summary>
+        /// <param name="input">An instance of an enum.</param>
+        /// <returns>True, if the instance maps directly to a single enum property, false if otherwise.</returns>
+        private static bool DirectlyMapsToEnumConstant(Enum input)
```

Comment:
A little bit late, but. Shouldn't it be `[Enum.IsDefilend](https://msdn.microsoft.com/en-us/library/system.enum.isdefined(v=vs.110).aspx)`?